lib/Support/assert.cpp

```diff
@@ -23,11 +23,14 @@ void llvm_assert(const char *_Message, const char *_File, unsigned _Line,
 
 #else
 
-#include <assert.h>
+#include "llvm/Support/Compiler.h"
+#include "llvm/Support/raw_ostream.h"
 
-void llvm_assert(const char *message, const char *, unsigned,
+void llvm_assert(const char *_Message, const char *_File, unsigned _Line,
                  const char *_Function) {
```

Comment:
nit: Since we're touching this might as well match LLVM style.

```suggestion
void llvm_assert(const char *Message, const char *File, unsigned Line,
                 const char *Function) {
```

Reply:
Done. Also updated the header file.